api/jobs/gears.py

```diff
@@ -119,12 +132,17 @@ def insert_gear(doc):
     if doc.get("invocation-schema"):
         del(doc["invocation-schema"])
 
+    now = datetime.datetime.utcnow()
+
+    doc['created']  = now
+    doc['modified'] = now
+
     config.db.gears.insert(doc)
```

Comment:
The job docs are still an ID, and I didn't want to hack on #654 until we make a branch just for that.